src/cassandra_statement.rs

```diff
@@ -1296,22 +1303,45 @@ mod tests {
 
     #[test]
     fn test_invalid_statement() {
-        let stmt = "This is an invalid statement";
-        let ast = CassandraAST::new(stmt);
+        let statement = "This is an invalid statement";
+        let ast = CassandraAST::new(statement);
+        assert!(ast.has_error());
+        let (result, parsed, _start, _end) = &ast.statements[0];
```

Comment:
A single assert_eq against the entire value of ast.statements[0] would make for a really robust test:

Something like:
```rust
assert_eq!(
    &ast.statements[0],
    &ParseStatement {
        error: false
        statement: CassandraStatement::Unknown(statement.to_string())
        bytes_parsed_start: 0,
       bytes_parsed_end: 10,
   }
)
```